src/core/server/config/env.ts

```diff
@@ -138,6 +139,7 @@ export class Env {
       buildNum: isKibanaDistributable ? pkg.build.number : Number.MAX_SAFE_INTEGER,
       buildSha: isKibanaDistributable ? pkg.build.sha : 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
       version: pkg.version,
+      dist: isKibanaDistributable,
```

Comment:
Although we used `dist` in LP what do you think of using a more descriptive key, we could use the full word `distributable` but `isKibanaDistributable` is probably even more descriptive like:
```ts
this.packageInfo = Object.freeze({
...
version: pkg.version,
isKibanaDistributable,
...
});
```

Reply:
I wish we did, but suspect it slows down migration as require manual changing. I'd rename this flag when plugins migrated to NP and implemented with Typescript.

Reply:
OK, how and where we introduce breaking changes probably warrants a bigger discussion and doesn't need to block this pr.